### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by tarsius**

### CODE REVIEW

File: diff-hl.el [hunk 1]
```diff
@@ -544,7 +544,7 @@ The value of this variable is a mode line template as in
         ;; https://github.com/magit/magit/issues/603
         (add-hook 'magit-revert-buffer-hook 'diff-hl-update nil t)
         ;; Magit versions 2.0-2.3 don't do the above and call this
-        ;; instead, but only when they dosn't call `revert-buffer':
+        ;; instead, but only when they doesn't call `revert-buffer':
```
Comment (dgutov) [inline diff-hl.el]: `don't`?

Comment (tarsius) [reply diff-hl.el]: Ah grab... :blush:

Comment (tarsius) [reply diff-hl.el]: Fixed fix.

**Comment by dgutov** [comment]
Thanks!

**Pull request merged**